src/test/java/org/apache/datasketches/kll/KllFloatsSketchTest.java

```diff
@@ -608,6 +610,101 @@ public void checkSortedViewAfterReset() {
     try { sk.getSortedView(); fail(); } catch (SketchesArgumentException e) { }
   }
 
+  @Test
+  public void checkVectorUpdate() {
+    boolean withLevels = false;
+    boolean withLevelsAndItems = true;
+    int k = 20;
+    int n = 108;//108;
+    int maxVsz = 40;  //max vector size
+    KllFloatsSketch sk = KllFloatsSketch.newHeapInstance(k);
+    int j = 1;
+    int rem;
+    while ((rem = n - j + 1) > 0) {
+      int vecSz = min(rem, maxVsz);
+      float[] v = new float[vecSz];
+      for (int i = 0; i < vecSz; i++) { v[i] = j++; }
+      sk.update(v, 0, vecSz);
+    }
+    println(LS + "#<<< END STATE # >>>");
+    println(sk.toString(withLevels, withLevelsAndItems));
+    println("");
+  }
+
+  @Test
+  public void vectorizedUpdates() {
+    final int trials = 1;
+    final int M = 1; //number of vectors
+    final int N = 1000; //vector size
+    final int K = 256;
+    final float[] values = new float[N];
+    float vIn = 1.0F;
+    long totN = 0;
+    final long startTime = System.nanoTime();
+    for (int t = 0; t < trials; t++) {
+      final KllFloatsSketch sketch = KllFloatsSketch.newHeapInstance(K);
+      for (int m = 0; m < M; m++) {
+        for (int n = 0; n < N; n++) {
+          values[n] = vIn++;  //fill vector
+        }
+        sketch.update(values, 0, N); //vector input
+      }
+      totN = sketch.getN();
+      assertEquals(totN, M * N);
+      assertEquals(sketch.getMinItem(), 1.0F);
+      assertEquals(sketch.getMaxItem(), totN);
+      assertEquals(sketch.getQuantile(0.5), (float)(totN / 2.0), totN * PMF_EPS_FOR_K_256 * 2.0); //wider tolerance
+    }
+    final long runTime = System.nanoTime() - startTime;
+    println("Vectorized Updates");
```

Comment:
This test was proposed by Fan Yang, which is not a bad test.  He had it configured with 1000 trials and 1000 vectors of size 1000, which is way too cumbersome for a unit test. I reconfigured it to 1 trial of 1 vector of size 1000 for use as a unit test. I also added the informative print statements to make the output more understandable.  I purposely left this test in, in case we want to ever expand it, which is trivial to do.

Reply:
The same is true of the following test "nonVectorizedUpdates".  Printing both tests allows direct comparison between the two update modes.

Reply:
I also fixed the "\n" issues only in the classes that were already included in this PR

Reply:
I think I'd still generally favor moving it to characterization since printing this much info is much more appropriate there but certainly not a hill I'm willing to die on. Or even get a splinter protecting. This is fine.